src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  function clear() {
+    for (const key in state) {
+      delete state[key];
+    }
+  }
+
+  function removeSomeKeys(keysArray) {
```

Comment:
this way it is clear enough
```suggestion
  function removeKeys(keysArray) {
```